iosApp/iosApp/Pages/NearbyTransit/NearbyViewModel.swift

```diff
@@ -63,7 +63,7 @@ class NearbyViewModel: ObservableObject {
     }
 
     func goBack() {
-        navigationStack.removeLast()
+        _ = navigationStack.popLast()
```

Comment:
I finally encountered this in the wild from hitting the back button twice quickly! Excited for this fix 🎉 
Consider adding a test for calling goBack on an empty list to prevent a regression